sdk/src/lib.rs

```diff
@@ -773,7 +773,7 @@ impl<T: Debug> WfExitValue<T> {
 
 /// Activity functions may return these values when exiting
 #[derive(derive_more::From)]
-pub enum ActExitValue<T: Debug> {
+pub enum ActExitValue<T: AsJsonPayloadExt> {
```

Comment:
It took me some time to figure this out, but we need some kind of bound here to help the trait solver disambiguate between `core`'s blanket identity `From` impl and the more specific one derived here. Given that `ActExitValue` is specifically used for `IntoActivityFunc` which requires this bound anyway, it seems sensible to just impose that directly on the type.

(Alternatively, replacing the derived `From` impl with a custom one that requires the `AsJsonPayloadExt` bound also works.)